frictionless_ckan_mapper/ckan_to_frictionless.py

```diff
@@ -1,118 +1,123 @@
+'''
+Convert CKAN packages and resources to their
+Frictionless equivalent.
+'''
+# Standard library imports
 import json
 
+# Third-party library imports
 import slugify
 
+try:
```

Comment:
This is needed for Python 2, where it throws an `AttributeError` instead of `json.decoder.JSONDecodeError`.